src/makeCalculator.js

```diff
@@ -5,6 +5,39 @@
  */
 function makeCalculator() {
   // write code here
+  const calc = {
+    result: 0,
+
+    add(objResult, num) {
+      return objResult + num;
+    },
+
+    subtract(objResult, num) {
+      return objResult - num;
+    },
+
+    multiply(objResult, num) {
+      return objResult * num;
+    },
+
+    divide(objResult, num) {
+      return objResult / num;
+    },
```

Comment:
[CODE KNOWLEDGE] - When creating methods in an object, you don't need to use the function keyword, use the method shorthand instead.